### ISSUE TITLE
connection error -

### ISSUE THREAD

**Issue by thestrayparticle** [m00nbyt3/SpoofyMailer#1]
Get an error - 
ERROR > Connection attempt to in-v3.mailjet.com:25 failed:IO::Socket::INET6: connect: connection refused

is it that here is no default smtp configured and I need to configure my own?

**Comment by m00nbyt3** [comment]
Yep, looks like preconfigured SMTP was banned, so you may have to use another one.
